src/graphql/generated.ts

```diff
@@ -8405,7 +8405,7 @@ export type DomainFragment = { __typename?: 'Domain', id: string, nativeId: numb
 
```

Comment:
There are no schema or *.graphql files changes, this file shouldn't have been modified